docs/source/jwt_endpoints.rst

```diff
@@ -16,9 +16,9 @@ Use this endpoint to obtain JWT.
 |          |                                 | * ``access``                     |
 |          |                                 | * ``refresh``                    |
 |          |                                 |                                  |
-|          |                                 | ``HTTP_401_UNAUTHORIZED``         |
+|          |                                 | ``HTTP_401_UNAUTHORIZED``        |
 |          |                                 |                                  |
-|          |                                 |  * ``non_field_errors``          |
+|          |                                 |  * ``detail``                    |
```

Comment:
Shouldn't this be `{{ api_settings.NON_FIELD_ERRORS_KEY }}`?

Reply:
It should, if the content was actually `api_settings.NON_FIELD_ERRORS_KEY`, but when I tested it, I only ever got a `detail` field in the data body (and sometime `code`), but never `non_field_errors`

Reply:
:thinking: I'll investigate it but won't hold your PR for this reason. Thanks for your work :+1: